distributed/diagnostics/nvml.py

```diff
@@ -105,7 +110,7 @@ def init_once():
             return
 
         if _in_wsl() and parse_version(
-            pynvml.nvmlSystemGetDriverVersion().decode()
+            _maybe_decode(pynvml.nvmlSystemGetDriverVersion())
```

Comment:
Would suggest using Dask's existing utility function for this, [`ensure_unicode`]( https://github.com/dask/dask/blob/a9396a913c33de1d5966df9cc1901fd70107c99b/dask/utils.py#L1116 )

To do this would add the following top-level `import`...

```python
from dask.utils import ensure_unicode
```

...then this line would become...

```suggestion
            ensure_unicode(pynvml.nvmlSystemGetDriverVersion())
```

Reply:
Note that there is a long history of doing this kind of thing: https://github.com/dask/distributed/pull/6295